src/commands/reminder/CheckReminder.ts

```diff
@@ -0,0 +1,50 @@
+import BaseCommand, { ICommandContext, } from "../BaseCommand";
+
+import { GeneralUtilities } from "../../utilities";
+// import { ButtonBuilder, ButtonStyle, ColorResolvable, TextInputBuilder, TextInputStyle } from "discord.js";
+import { PostGresReminder as PostGresReminder } from "../../utilities/PostGresReminder";
+
+export default class CheckReminder extends BaseCommand {
+    public constructor() {
+        super ({
+            cmdCode: "CHECKREMINDER",
+            formalCommandName: "CheckReminder",
+            botCommandName: "checkreminder",
+            description: "Check reminders.",
+            generalPermissions: [],
+            botPermissions: [],
+            argumentInfo: [],
+            commandCooldown: 4 * 1000,
+            guildOnly: false,
+            botOwnerOnly: false,
+        });
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public async run(ctx: ICommandContext): Promise<number> {
+        const messages = await PostGresReminder.searchByUser(ctx.user.id);
+        let fmt_msg = "\n";
```

Comment:
We use camel case for variable naming, not snake case.
```suggestion
        let fmtMsg = "\n";
```